### ISSUE TITLE
Delete button on forms index page not clickable

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#919]
**Description**
The delete button on the forms index page is not clickable.

![image](https://user-images.githubusercontent.com/16959207/171198244-06ce2e10-213c-49b6-b487-412938eb65b1.png)

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Both
- Client-side Validation: Both
- Custom Form Templates: No

**Additional info**
- Plugin version: 2.0.0-beta.10
- Craft version: 4.0.3 Pro
- Multi-site: No

**Comment by mtnorthrop** [comment]
I'm also using Formie 2.0.0-beta.10 on Craft 4.0.3 and am seeing the same issue when trying to delete a submission from the submissions index, trying to permanently delete a spam submission, and trying to permanently delete a trashed submission. (I'm guessing these are all due to the same underlying issue, along with the main issue reported above.)

**Delete from submissions index:**
![screenshot-vwh-website ddev site_44301-2022 05 31-17_06_17](https://user-images.githubusercontent.com/800546/171207722-53e8036e-7f38-4cc2-8160-8bea6aaccd95.png)

**Permanently delete spam submission:**
![screenshot-vwh-website ddev site_44301-2022 05 31-17_07_19](https://user-images.githubusercontent.com/800546/171207875-7cf33b4f-b87d-4f6a-911e-5079311fccca.png)

**Permanently delete trashed submission:**
![screenshot-vwh-website ddev site_44301-2022 05 31-17_08_10](https://user-images.githubusercontent.com/800546/171207957-5f360922-b991-4180-901c-755ccd4f8749.png)

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.11](https://github.com/verbb/formie/releases/tag/2.0.0-beta.11)
